src/core/sys/posix/sys/types.d

```diff
@@ -1056,37 +1056,40 @@ else version( CRuntime_Bionic )
         size_t  guard_size;
         int     sched_policy;
         int     sched_priority;
-        version(AArch64) char[16] __reserved;
+        version(D_LP64) char[16] __reserved;
     }
 
     struct pthread_cond_t
     {
-        int value; //volatile
+        version(D_LP64)
+            int[12] __private;
+        else
+            int[1] __private;
     }
 
     alias c_long pthread_condattr_t;
     alias int    pthread_key_t;
 
     struct pthread_mutex_t
     {
-        int value; //volatile
+        version(D_LP64)
+            int[10] __private;
+        else
+            int[1] __private;
     }
 
     alias c_long pthread_mutexattr_t;
-    alias int    pthread_once_t; //volatile
+    alias int    pthread_once_t;
 
     struct pthread_rwlock_t
     {
-        pthread_mutex_t  lock;
-        pthread_cond_t   cond;
-        int              numLocks;
-        int              writerThreadId;
-        int              pendingReaders;
-        int              pendingWriters;
-        void*[4]         reserved;
```

Comment:
Isn't this a breaking change as people might have been accessing these fields?

Reply:
Yep, [they chose to break it 3 years back](https://android.googlesource.com/platform/bionic/+/2fabea47ac9475bcc52aff0715819d18aa5bdf1d%5E%21/#F1), just updating this file to reflect that change.

Reply:
Could we add forwarding properties and deprecate those?

Reply:
Eh, I doubt anyone is using this Bionic declaration, and given that it's broken upstream in their libc, what's the point?

Reply:
According to the Posix standard, the fields of those structs are implementation defined, so the user has no business in acessing them.

Reply:
For libc implementations we only need to ensure that they have the same ABI (size and perhaps alignment).